packages/devtools_server/lib/src/server.dart

```diff
@@ -276,6 +282,16 @@ Future<void> registerLaunchDevToolsService(
   }
 }
 
+bool isVersionLessThan(
```

Comment:
add a TODO that this is copied from DevTools and that this util method should be upstreamed to the vm_service library.

Reply:
Done!